packages/atlas/src/providers/wallet/wallet.provider.tsx

```diff
@@ -42,6 +43,41 @@ export const WalletProvider: FC<PropsWithChildren> = ({ children }) => {
     [_setWalletAccounts]
   )
 
+  const getWalletConnectParams = useCallback(async () => {
+    const chainMetadata = await joystreamCtx?.joystream?.getChainMetadata()
+
+    const genesisHash = chainMetadata?.genesisHash
+
+    if (!genesisHash) {
+      SentryLogger.error(
+        'WalletConnect Config error',
+        'wallet.provider.tsx',
+        'No genesis hash found in chain metadata. WalletConnect configuration is incorrect'
+      )
+    }
+    const chainId = `polkadot:${genesisHash?.replace('0x', '').substring(0, 32)}` as `polkadot:${string}`
+
+    const walletConnectParams: WalletConnectConfiguration = {
+      projectId: atlasConfig.features.members.walletConnectProjectId || '',
+      relayUrl: 'wss://relay.walletconnect.com',
+      metadata: {
+        name: 'Joystream demo app',
+        description: 'Joystream Wallet-Connect',
+        url: '#',
+        icons: ['https://walletconnect.com/static/favicon.png'],
```

Comment:
Shouldn't this be Atlas/Gleev logo? My understanding is that this is passed to the wallet?